lightly/active_learning/scorers/keypoint_detection.py

```diff
@@ -0,0 +1,91 @@
+from typing import List, Dict
+
+import numpy as np
+
+from lightly.active_learning.scorers import Scorer
+from lightly.active_learning.utils.keypoint_detection_output import \
+    KeypointDetectionOutput, KeypointDetection
+
+
+def _least_confidence(
+        model_output: List[KeypointDetectionOutput]) -> np.ndarray:
+    """Score which prefers samples with low confidence score.
+    
+    The confidence score per image is 1 minus the mean confidence
+    score of all its keypoints.
+
+    Args:
+        model_output:
+            Predictions of the model of length N.
+
+    Returns:
+        Numpy array of length N with the computed scores.
+
+    """
+    scores = []
+    for keypoint_detection_output in model_output:
+        confidences_this_detection = []
```

Comment:
this name is very confusing, don't you mean something like `confidences_per_image` and then below the `confidences` are `confidences_per_instance`?